test/jmh/BUILD

```diff
@@ -0,0 +1,6 @@
+load("//jmh:jmh.bzl", "scala_benchmark_jmh")
+
+scala_benchmark_jmh(
```

Comment:
can we add to the `run_test.sh` script to actually run a benchmark to make sure things run, not just build?